react-app/ccf/src/pages/admin-post-grant-reports/AdminPostGrantReports.css

```
@@ -86,51 +93,55 @@
+.reports-table-wrapper {
+    background-color: white;
+    border: 2px solid #BE0019;
+    border-radius: 8px;
     overflow: hidden;
-    margin-bottom: 30px;
-    border: 1px solid #e9ecef;
 }
 
 .reports-table {
     width: 100%;
     border-collapse: separate;
```

Stylelint (17.7.0)

[error] Duplicate property "border-collapse" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)